pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -239,6 +244,11 @@ func (lrw *logicalReplicationWriterProcessor) Start(ctx context.Context) {
 		}
 		return nil
 	})
+
+	if err := lrw.dlqClient.Create(); err != nil {
```

Comment:
i think we'll want to create the DLQ table on the job coordinator, not on each proc, like around here https://github.com/msbutler/cockroach/blob/butler-skip-auto-replan-stress/pkg/ccl/crosscluster/logical/logical_replication_job.go#L74

Reply:
like there may be multiple procs replicating one table, so it seems kinda weird to attempt to create the dlq on each proc.